### ISSUE TITLE
TypeError: t.toFloat is not a function

### ISSUE THREAD

**Issue by rafidgotit** [bensonruan/Face-Mask#1]
I was trying to understand how Tensorflow wroks on web. As your instruction I cloned your project and opened it on my browser. I faced the issue.
`Uncaught (in promise) TypeError: t.toFloat is not a function
    c https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    tidy https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core:17
    scopedRun https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core:17
    tidy https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core:17
    kn https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core:17
    estimateFaces https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    a https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    a https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    r https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    r https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    estimateFaces https://cdn.jsdelivr.net/npm/@tensorflow-models/facemesh:17
    detectFaces https://localhost/face/js/face-mask.js:109
    <anonymous> https://localhost/face/js/face-mask.js:77
    promise callback* https://localhost/face/js/face-mask.js:75`

**Comment by KaranMonani** [comment]
This issue can be solved by downgrading tensorflow version to 2.6.0

Replace below CDN in Index.html

```
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core@2.6.0/dist/tf-core.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-converter@2.6.0"></script>
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgl@2.6.0"></script>
```

that should work

**Comment by rafidgotit** [comment]
> 
> 
> This issue can be solved by downgrading tensorflow version to 2.6.0
> 
> Replace below CDN in Index.html
> 
> ```
> <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core@2.6.0/dist/tf-core.min.js"></script>
> <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-converter@2.6.0"></script>
> <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgl@2.6.0"></script>
> ```
> 
> that should work

Thank you so much!
